openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -2489,12 +2546,117 @@ def check_handler_compatibility(self,
                                          tolerance_attrs=float_attrs_to_compare+unit_attrs_to_compare,
                                          tolerance=self._SCALETOL)
 
+
+    def assign_charge_from_molecules(self, molecule, charge_mols):
```

Comment:
Previously, `ToolkitAM1BCCHandler` was responsible for assigning charges from molecules in the `charge_from_molecules` kwarg. However, it makes more sense to have that functionality in the `ElectrostaticsHandler`, since the addition of LibraryCharges means that we won't always a `ToolkitAM1BCCHandler` in the `ForceField`.